### PR TITLE
fix(orbital): Fix event sampling and Safari SSE connection dropping

### PR DESCRIPTION
**Pull request by rahulchhabria**
## Summary

Two bugs were causing events to be dropped or stop rendering entirely.

### Clock skew drops live events
The stale-burst guard used `Math.abs()` which dropped events in both directions. Source servers with clocks ~5s ahead of the client were causing live events to be incorrectly discarded. Fixed to only drop events that are genuinely old (>10s in the past) — stale SSE bursts always arrive with past timestamps, not future ones.

### Safari stops rendering after ~30s
Three compounding issues caused Safari to silently stop receiving events:

- **`visibilitychange` stuck** — Safari iOS sometimes fires `pagehide` when switching apps but skips the matching `visibilitychange` on return, leaving `windowInFocus = false` permanently. Added `pageshow`/`pagehide` listeners as reliable Safari fallbacks.
- **SSE reconnect silently fails** — Safari fires `onerror` with `readyState=CONNECTING` but never actually reconnects. The old handler only retried on `CLOSED`. Now force-reconnects on both states with a clean teardown.
- **No last-resort recovery** — Added a 30s watchdog timer that resets on every received SSE message and force-reconnects if the stream goes silent, catching edge cases where the connection dies without firing an error event.

## Test plan

- [ ] Open in Safari — events should continue flowing past 30s without interruption
- [ ] Switch apps on iOS Safari and return — events should resume immediately
- [ ] In production with a source server clock ahead by ~5s — no "Dropping events" warnings in the console
- [ ] Background the tab and foreground it — no stale burst warnings, events resume cleanly

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: static/orbital.js [hunk 1]
```diff
@@ -384,7 +384,32 @@ function addFeedItem(platform, lat, lng) {
...
+    console.warn('[Sentry Live] No events for 30s — forcing SSE reconnect');
+    Sentry.addBreadcrumb({ category: 'sse', message: 'watchdog triggered reconnect', level: 'warning' });
+    if (source) {
+      source.onmessage = null;
+      source.onerror   = null;
+      source.close();
+      source = null;
+    }
+    connectStream();
+  }, 30000);
```
Comment (BYK) [inline static/orbital.js]: 30 seconds is a very long time for Sentry events. I'd make it 5 to keep responsiveness.

File: main.go [hunk 2]
```diff
@@ -240,6 +240,16 @@ func main() {
 	}
 	defer conn.Close()
 	rng := rand.New(rand.NewSource(time.Now().UnixNano()))
+	// Heartbeat: send a ping every 20s so the client watchdog doesn't
+	// fire during legitimate low-traffic periods.
+	go func() {
+		ticker := time.NewTicker(20 * time.Second)
```
Comment (BYK) [inline main.go]: Accordingly, this should be around 5 seconds or less

**Review by BYK — APPROVED** [review]
